symplyphysics/laws/thermodynamics/maxwell_boltzmann_distributions/speed_distribution.py

```diff
@@ -27,10 +27,10 @@
 ## - Particles are identical, non-interacting, non-relativistic, and obeying classical laws of physics.
 ## - The ensemble of particles is at thermodynamic equilibrium.
 
-speed_distribution_function = Symbol("speed_distribution_function", 1 / units.velocity)
-particle_speed = Symbol("particle_speed", units.velocity)
-particle_mass = clone_symbol(symbols.basic.mass, "particle_mass")
-equilibrium_temperature = clone_symbol(symbols.thermodynamics.temperature, "equilibrium_temperature")
+speed_distribution_function = Symbol("speed_distribution_function", 1 / units.velocity, positive=True)
+particle_speed = Symbol("particle_speed", units.velocity, nonnegative=True)
```

Comment:
Maybe use `positive` instead?

Reply:
But it can be 0

Reply:
Temperature can also be 0.

Reply:
Then it makes sense to mark temperature as `nonnegative` as well. But the number of cases where we have to deal with zero absolute temperature is, in my opinion, much smaller than those with zero speed

Reply:
I want these assumptions to be consistent all over the code. So if we use `nonnegative` parameter, every `positive` should be updated to it. There are many.

Reply:
Well now I agree that in this very formula it's more rational to have `positive` since the RHS is positive (unless `T` is zero but that's rare). Btw most of the codebase (95%-ish) doesn't use any assumptions at all (and I guess we should add them)

Reply:
You can take on this task actually - start adding assumptions all over the code, including fixing the `positive` assumption. It's a big, but relatively easy task.

Reply:
The reason I add them is because `sympy` doesn't otherwise want to solve or simplify equations the way we need them because all the symbols are `complex` unless stated otherwise. Specifically in these laws where we're dealing with the speed distribution function.

For example, this is what happens in the law of the mean square of speed:
![image](https://github.com/blackyblack/symplyphysics/assets/115647936/474ffbb0-6317-4b45-88f5-429c9ea12e68)

Which immediately simplifies to the following once the correct assumptions are added:
![image](https://github.com/blackyblack/symplyphysics/assets/115647936/543a24e4-2b1a-4180-ab62-280635df43b5)

Reply:
Yes, I understand it. But for the sake of making proof work, we could add assumptions as soon as we are trying to prove it. When we start adding assumption right in the laws, we are making them a part of those laws. Now we should look carefully, if law makes sense for some negative parameters, for example.

Reply:
For physcial laws I expect most variables to be `real` and `nonnegative`.